pyproject.toml

```diff
@@ -44,4 +44,5 @@ testpaths = [
     "src/lando/dockerflow",
     "src/lando/main/tests.py",
     "src/lando/tests",
+    "src/lando/ui/tests/test_forms.py",
```

Comment:
Please add a `TODO` comment linking to the bug here.

Reply:
What value will that add? We are already tracking this bug on bugzilla and it doesn't affect functionality. On line 45 we also have a single module as a test file, so this is not something unusual either.

Reply:
Leaving a comment in the code that points to the bug provides immediate context to future developers or reviewers, making it clear why certain tests are not enabled. It helps maintain transparency, ensures the issue isn’t forgotten, and avoids confusion when someone revisits the code later.

Reply:
I don't agree that this is necessary in this case, but I agree that it can be, in some cases.

I think in most cases we should not be referencing bugs in the code. Bugs are mutable after all, and you can't know if a bug is referenced in the code base before you change it.

Reply:
Regardless I will add it to this PR since this seems important to you at this stage.